tests/async_action.cpp

```diff
@@ -0,0 +1,102 @@
+#include "async_action.h"
+#include "ast/async_event_types.h"
+#include "bpftrace.h"
+#include "mocks.h"
+#include "types.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace bpftrace::test::async_action {
+
+using namespace bpftrace::async_action;
+
+TEST(asyncaction_handler, join)
```

Comment:
```suggestion
TEST(async_action_handler, join)
```
And for the rest of them, please 🙏

Reply:
Done